zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/AuthorizationCheckBehavior.java

```diff
@@ -352,9 +352,14 @@ private Stream<String> getMappingsAuthorizedResourceIdentifiers(
                       request.getResourceType(),
                       request.getPermissionType())
                   .forEach(stream);
-              getAuthorizedResourceIdentifiersForOwnerKeys(
+              getAuthorizedResourceIdentifiersForOwners(
                       AuthorizationOwnerType.ROLE,
-                      mapping.getRoleKeysList(),
+                      membershipState.getMemberships(
+                          EntityType.MAPPING,
+                          // TODO: Use mappingId instead of mappingKey when roles and mappings are
```

Comment:
❓ Is this a known place to follow up after the other tasks in the related epics?

Reply:
I'd hope so :sweat_smile: I think ultimately we should remove the mapping key altogether so this would break compilation.